ipa.go

```diff
@@ -69,16 +72,6 @@ func FromBytes(fr *Fr, data []byte) {
 	fr.SetBytes(aligned[:])
 }
 
-func StemFromBytes(fr *Fr, data []byte) {
-	bytes := make([]byte, len(data))
-	copy(bytes, data)
-	fr.SetBytesLE(bytes)
-}
-
 func Equal(self *Point, other *Point) bool {
 	return other.Equal(self)
 }
-
-func Generator() *Point {
-	return new(Point).Identity()
-}
```

Comment:
This isn't necessary since we merged [this PR](https://github.com/gballet/go-verkle/pull/340/files#diff-050830ac4334170e8d335e1b95519052a3461c5b57a4ca38a0531e35a8f3d388L212-L213).